docusaurus.config.js

```diff
@@ -125,6 +125,14 @@ const config = {
           {
             title: 'Community',
             items: [
+              {
+                label: 'Slack - Podman Community',
+                href: 'https://app.slack.com/client/T08PSQ7BQ/C08MXJLCFCN',
+              },
+              {
+                label: 'Podman Development',
+                href: 'https://app.slack.com/client/T08PSQ7BQ/C08NTKCDC1W',
+              },
```

Comment:
Looks like this is missing the matrix link, while it is pre existing I like to say the actual community page and the footer should be in sync